### ISSUE TITLE
Scene produces spam of Non-human-readable errors

### ISSUE THREAD

**Issue by DataPlusProgram** [godotengine/godot#44205]
**Godot version:**
3.2.3.stable.official

**OS/device including version:**
Windows 10

**Issue description:**
Opening this scene will create an error spam of:

```
ERROR: build: Condition "!F" is true. Continuing.
   At: core/math/quick_hull.cpp:399
ERROR: build: Condition "O == __null" is true. Continuing.
   At: core/math/quick_hull.cpp:402
```

The meshes for this scene were created programmatically using "create_trimesh_collision()"

Everything appears to work correctly besides the constant error messages which cause slowdown on larger scenes 

The error codes are non-human-readable making it hard to know what is going on or how to fix it

**Steps to reproduce:**

Open the provided scene

**Minimal reproduction project:**

[testProject.zip](https://github.com/godotengine/godot/files/5662613/testProject.zip)

**Comment by kleonc** [comment]
Present in `3.3.2.stable`.
Fixed in `3.4.beta3` (most likely by #48533).

**Comment by Calinou** [comment]
Closing per @kleonc's comment.
